index.html

```diff
@@ -534,51 +547,67 @@ <h2>The <dfn>WakeLockSentinel</dfn> interface</h2>
           attribute EventHandler onrelease;
         };
       </pre>
-      <p>A <a>WakeLockSentinel</a> object provides a handle to a <a>platform
-      wake lock</a>, and it holds on to it until it is either manually released
-      or until the underlying <a>platform wake lock</a> is released. Its
-      existence keeps a <a>platform wake lock</a> for a given <a>wake lock
-      type</a> active, and releasing all
-      <a>WakeLockSentinel</a> instances of a given <a>wake lock type</a> will
-      cause the underlying <a>platform wake lock</a> to be released.
+      <p>
+        A {{WakeLockSentinel}} object provides a handle to a <a>platform wake
+        lock</a>, and it holds on to it until it is either manually released or
+        until the underlying <a>platform wake lock</a> is released. Its
+        existence keeps a <a>platform wake lock</a> for a given <a>wake lock
+        type</a> active, and releasing all {{WakeLockSentinel}} instances of a
+        given <a>wake lock type</a> will cause the underlying <a>platform wake
+        lock</a> to be released.
+      </p>
       <aside class="note">
         See <a>auto-releasing wake locks</a>, <a>handling document loss of full
         activity</a> and <a>handling document loss of visibility</a> for
         circumstances under which a given wake lock may be released by the
         <a>user agent</a>.
       </aside>
-      <p>The <dfn>type</dfn> attribute corresponds to the
-      <a>WakeLockSentinel</a>'s <a>wake lock type</a>.</p>
+      <p>
+        The <dfn>type</dfn> attribute corresponds to the {{WakeLockSentinel}}'s
+        <a>wake lock type</a>.
+      </p>
       <section>
-        <h3>The <dfn>release()</dfn> method</h3>
-        <p>The <a>release()</a> method, when invoked, MUST run the following
-        steps:</p>
+        <h3>
+          The <dfn>release()</dfn> method
+        </h3>
+        <p>
+          The {{WakeLockSentinel/release()}} method, when invoked, MUST run the
+          following steps:
+        </p>
         <ol class="algorithm">
-          <li>Let |promise:Promise| be <a>a new promise</a>.</li>
+          <li>Let |promise:Promise| be <a>a new promise</a>.
```

Comment:
Similarly, I can't seem to find anything in ReSpec's documentation that describes the difference between `|foo|`, `<a>foo</a>`, `{{foo}}` and when to use each.

Reply:
So, |foo| is a variable.  {{foo}} is "this is IDL". and `<a>` is usually a concept link, but works with IDL for backwards compat.

Reply:
Got it, thanks for adding that to the ReSpec wiki as well.